deps/rabbit/src/rabbit_amqp_management.erl

```diff
@@ -147,29 +146,36 @@ handle_http_req(HttpMethod = <<"PUT">>,
             Result;
         {error, not_found} ->
             PermCache2 = check_dead_letter_exchange(QName, QArgs, User, PermCache1),
-            case rabbit_amqqueue:declare(
-                   QName, Durable, AutoDelete, QArgs, Owner, Username) of
-                {new, Q} ->
+            try case rabbit_amqqueue:declare(
```

Comment:
Ideally, `rabbit_amqqueue:declare/6` would return `{protocol_error, _ErrorType, Reason, ReasonArgs}` instead of additionally throwing.

Reply:
For now, we can catch. Only `rabbit_amqqueue:declare/6` will throw, so only that part should be protected. Remove the inner `case` statement.

Reply:
I resolved this.

Reply:
Thank you